wallet/src/money.rs

```diff
@@ -0,0 +1,112 @@
+//! Wallet features related to spending money and checking balances.
+
+use crate::{fetch_storage, SpendArgs};
+
+use std::{thread::sleep, time::Duration};
+
+use jsonrpsee::{core::client::ClientT, http_client::HttpClient, rpc_params};
+use parity_scale_codec::{Decode, Encode};
+use runtime::{
+    money::{Coin, MoneyVerifier},
+    OuterRedeemer, OuterVerifier, Transaction,
+};
+use sp_core::{crypto::Pair as PairT, sr25519::Pair, H256};
+use sp_runtime::traits::{BlakeTwo256, Hash};
+use tuxedo_core::{
+    redeemer::SigCheck,
+    types::{Input, Output, OutputRef},
+};
+
+/// Create and send a transaction that spends coins on the network
+pub async fn spend_coins(client: &HttpClient, args: SpendArgs) -> anyhow::Result<()> {
+    println!("The args are:: {:?}", args);
+    let (provided_pair, _) = Pair::from_phrase(&args.seed, None)?;
+
+    // Construct a template Transaction to push coins into later
+    let mut transaction = Transaction {
+        inputs: Vec::new(),
```

Comment:
Only nit I would say is sometimes in the code we initialize with ‘Vec::new()’ and other times with ‘vec![]’ we should probably choose one for consistency

Reply:
Agreed. Do you have a preference?